modules/acquia_cms_search/composer.json

```diff
@@ -45,8 +45,7 @@
         "enable-patching": true,
         "patches": {
             "drupal/facets": {
-                "3259123 - ViewsDisplayBase::isRenderedInCurrentRequest() doesn't let the facet link to reset to original URL": "https://www.drupal.org/files/issues/2022-01-18/3259123-search-facet-reset-link.patch",
-                "3358295 - PHP 8.2 compatibility": "https://git.drupalcode.org/project/facets/-/merge_requests/141.patch"
```

Comment:
Note: while reviewing this issue, kindly check [this](https://www.drupal.org/project/facets/issues/3360426) issue